sql/2025-04-30-drop-namespace-diffs.sql

```diff
@@ -0,0 +1,3 @@
+-- Delete all previously-computed namespace diffs, because the diff payload is different now (we explicitly store
+-- errors).
+TRUNCATE namespace_diffs;
```

Comment:
Depending on whether you choose to re-queue diffs when the page is loaded and it's not cached, you may want to repopulate the queue with all existing contribution diffs after wiping them out.

If you have it set to requeue on page load then no need